src/Magento/FunctionalTestingFramework/Composer/Handlers/AbstractComposerHandler.php

```diff
@@ -0,0 +1,102 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\FunctionalTestingFramework\Composer\Handlers;
+
+use Composer\Package\CompletePackageInterface;
+use Magento\FunctionalTestingFramework\ObjectManager;
+use Magento\FunctionalTestingFramework\Composer\Objects\ComposerFactory;
+
+/**
+ *  Abstract Composer Handler
+ */
+abstract class AbstractComposerHandler
+{
+    /**
+     * Mftf test module type
+     */
+    const TEST_MODULE_PACKAGE_TYPE = 'magento2-functional-test-module';
+
+    /**
+     * Magento module type
+     */
+    const MAGENTO_MODULE_PACKAGE_TYPE = 'magento2-module';
+
+    /**#@+
+     * Composer package array keys
+     */
+    const KEY_PACKAGE_NAME = 'name';
+    const KEY_PACKAGE_TYPE = 'type';
+    const KEY_PACKAGE_VERSION = 'version';
+    const KEY_PACKAGE_DESCRIPTION = 'description';
+    const KEY_PACKAGE_INSTALLEDPATH = 'installedPath';
+    const KEY_PACKAGE_REQUIRES = 'requires';
+    const KEY_PACKAGE_DEVREQUIRES = 'devRequires';
+    const KEY_PACKAGE_SUGGESTS = 'suggests';
+    const KEY_PACKAGE_SUGGESTED_MAGENTO_MODULES = 'suggestedMagentoModules';
+    /**#@-*/
+
+    /**#@+
+     * Suggest field names
+     */
+    const SUGGEST_TYPE = 'type';
+    const SUGGEST_NAME = 'name';
+    const SUGGEST_VERSION = 'version';
+    /**#@-*/
+
+    /**
+     * @var \Composer\Composer
+     */
+    protected $composer;
+
+    /**
+     * @var \Composer\Package\Locker
+     */
+    protected $locker;
+
+    /**
+     * @var ComposerFactory
+     */
+    protected $composerFactory;
+
+    /**
+     * @var \Composer\Package\CompletePackage
+     */
+    protected $rootPackage;
+
+    /**
+     * @param ComposerFactory $composerFactory
+     */
+    public function __construct(ComposerFactory $composerFactory)
+    {
+        $this->composerFactory = $composerFactory;
```

Comment:
See comment in `ComposerFactory`, I think we can remove this class entirely.

Reply:
Done